tests/test_matcher.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+from pytest_httpserver import HTTPServer
+
+
+def test_expect_methods_are_equivalent(httpserver: HTTPServer):
+    expected_response = "OK"
+    matcher = httpserver.create_matcher(uri="/test", method="POST")
+    httpserver.expect(matcher).respond_with_data(expected_response)
+    resp = requests.post(httpserver.url_for("/test"), json={"list": [1, 2, 3, 4]})
+    assert resp.text == expected_response
+    httpserver.clear()
+
+    httpserver.expect_request("/test", method="POST").respond_with_data("OK")
+    resp = requests.post(httpserver.url_for("/test"), json={"list": [1, 2, 3, 4]})
+    assert resp.text == expected_response
```

Comment:
could you please remove these lines (12-16)? I think these are testing httpserver.expect_request with a json value, which is already tested here:

https://github.com/csernazs/pytest-httpserver/blob/de479838f6a0fb006fefa2f9cc37e50ae221d794/tests/test_json_matcher.py#L9-L15
```suggestion
```